src/components/burger-constructor/burger-constructor.tsx

```diff
@@ -0,0 +1,78 @@
+// @ts-nocheck
+
+import constructorStyle from './burger-constructor.module.css';
+import {ConstructorElement, Button, CurrencyIcon} from '@ya.praktikum/react-developer-burger-ui-components';
+import FillingIngredient from '../filling-ingredient/filling-ingredient'
+import PropTypes from 'prop-types';
+
+function BurgerConstructor(props) {
+
+  const {upside, fillings, downside} = {...props.burger};
```

Comment:
кажется тут использовоно лишнее создание объекта, тут сразу можно обратиться к `props.burger` без создания нового объекта
```suggestion
  const { upside, fillings, downside } = props.burger;
```